tests/Php80/Php80Test.php

```diff
@@ -177,6 +186,11 @@ public function testStrEndsWith()
         $testEmoji = '🙌🎉✨🚀'; // 0xf0 0x9f 0x99 0x8c 0xf0 0x9f 0x8e 0x89 0xe2 0x9c 0xa8 0xf0 0x9f 0x9a 0x80
         $this->assertTrue(str_ends_with($testEmoji, "🚀")); // 0xf0 0x9f 0x9a 0x80
         $this->assertFalse(str_ends_with($testEmoji, "✨")); // 0xe2 0x9c 0xa8
+
+        $this->assertTrue(str_starts_with(null, ''));
+        $this->assertTrue(str_starts_with('', null));
+        $this->assertFalse(str_starts_with(null, 'test'));
+        $this->assertTrue(str_starts_with('test', null));
```

Comment:
Please add some tests for type errors, when passing a bool or other types there.

Reply:
I've added these, are you happy @IonBazan?